_posts/en/pages/2017-01-01-download.md

```diff
@@ -54,6 +54,92 @@ patient: "Check your bandwidth and space"
 # releasekeys: "Bitcoin Core Release Signing Keys"
 releasekeys: "Bitcoin Core Release Signing Keys"
 
+### FIXME NEW TEXT
+
+pgp_key_fingerprint: "PGP key fingerprint"
+verify_download: "Verify your download"
+verification_recommended: "Download verification is optional but highly recommended.  Click one of the lines below to view verification instructions for that platform."
+windows_instructions: "Windows verification instructions"
+macos_instructions: "MacOS verification instructions"
+linux_instructions: "Linux verification instructions (not for Ubuntu PPA)"
+ubuntu_ppa_instructions: "Ubuntu PPA verification instructions"
+download_release: "Click the link in the list above to download the release for your platform and wait for the file to finish downloading."
+download_checksums: "Download the list of cryptographic checksums:"
+cd_to_downloads: "Open a terminal (command line prompt) and Change Directory (cd) to the folder you use for downloads.  For example:"
+cd_example_linux: "cd Downloads/"
+cd_example_windows: >
+  cd %UserProfile%\Downloads
+
+verify_download_checksum: "Verify the checksum of the release file is listed in the checksums file using the following command:"
+checksum_warning_and_ok: 'In the output produced by the above command, you can safely ignore any warnings and failures, but you must ensure the output lists "$(SHASUMS_OK)" after the name of the release file you downloaded.  For example:'
+obtain_release_key: "Obtain a copy of the release signing key by either clicking the following download link or by running the subsequent command:"
```

Comment:
I think it's better to remove the "download key" option. If you keep it you should also add instructions how to import the key from file.